src/com/teamcautionrobotics/robot2018/Robot.java

```diff
@@ -87,6 +90,27 @@ public void teleopPeriodic() {
             spinSpeed = manipulator.getAxis(Axis.RIGHT_TRIGGER);
         }
         intake.spin(spinSpeed);
+
+        boolean liftRaiseButton = manipulator.getButton(Button.Y);
+        if (liftRaiseButton != liftRaiseButtonPressed) {
+            if (liftRaiseButton) {
+                lift.setLevel(lift.getCurrentLiftLevel().next());
+            }
+            liftRaiseButtonPressed = liftRaiseButton;
+        }
+
+        boolean liftLowerButton = manipulator.getButton(Button.Y);
+        if (liftLowerButton != liftRaiseButtonPressed) {
+            if (liftLowerButton) {
+                lift.setLevel(lift.getCurrentLiftLevel().previous());
+            }
+            liftLowerButtonPressed = liftLowerButton;
+        }
+
+        // manual control
+        double manualLiftAxis = manipulator.getAxis(Axis.RIGHT_Y);
```

Comment:
`manualLiftAxis` is not an Axis (the axis is Y), but a command or some better term.